operations/iuf/workflows/slingshot_management_network_switch_updates.md

```diff
@@ -0,0 +1,31 @@
+# Perform Slingshot switch and management network switch firmware updates
+
```

Comment:
Would it be reasonable to note that switch firmware updates may cause temporary interrupts in network traffic during the upgrade procedure at the start of this section?